scripts/generate_test_epub.py

```
@@ -24,53 +32,59 @@
 SCREEN_WIDTH = 480
 SCREEN_HEIGHT = 800
 
+
 def get_font(size=20):
     """Get a font, falling back to default if needed."""
     try:
-        return ImageFont.truetype("/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf", size)
+        return ImageFont.truetype(
+            "/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf", size
+        )
     except:
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)